### PR TITLE
updated the typography and some other things

### PR DESCRIPTION
**Pull request by alitahir4024**
updated the typography of our website and also I've changed the body color as you wished,also updated the navbar and the nav menu according to the changes done

### CODE REVIEW

File: css/main.css [hunk 1]
```diff
@@ -25,6 +39,12 @@ html {
 
 .nav-menu-btn {
   background-color: transparent;
+  color: #495057;
+  transition: all 0.5s;
+}
+
+.onscroll-color {
+  color: #f5f5f5;
```
Comment (adilzulfiqar) [inline css/main.css:47]: [Suggestion] You can CSS variable here which will be a good approach to reuse properties. 

Reference:  https://www.w3schools.com/css/css3_variables.asp

Comment (alitahir4024) [reply css/main.css:47]: Thanks for pointing that out,actually it didn't pop out in my mind
will use it next time

Comment (adilzulfiqar) [reply css/main.css:47]: Cool

Comment (adilzulfiqar) [reply css/main.css:47]: Testing comment. Ignore it

**Pull request merged**